### DISCUSSION TITLE
Support security key auth on android

### DISCUSSION THREAD

**Discussion by alita-moore** [simple-login/app#807] [Feature Requests]
I noticed that the app doesn't support security key. But then when I try to use the API key work around it clears the API key every time I login. So I have to login on my browser to simplelogin.io and that supports the security key. 

Are there plans to get that feature out or should I just simple login in my browser from now on?

**Comment by nguyenkims** [comment]
@alita-moore yes we do have a plan to support security key in the mobile app.
